distributed/config.yaml

```diff
@@ -17,6 +17,20 @@ multiprocessing-method: forkserver
 # Communication options
 tcp-timeout: 30         # seconds delay before calling an unresponsive connection dead
 default-scheme: tcp
+require-encryption: False   # whether to require encryption on non-local comms
```

Comment:
Thanks, this looks good. Perhaps add a comment to the tail of the `ciphers` section to note the cipher string format?